mozregression/bisector.py

```diff
@@ -66,20 +70,26 @@ def initialize(self):
         This will only be called if there is some build data.
         """
         self.found_repo = self.build_data[0]['repository']
-        self.good_revision = self.build_data[0]['changeset']
-        self.bad_revision = self.build_data[-1]['changeset']
+        first, last = self._reverse_if_hunt_fix(0, -1)
+        self.good_revision = self.build_data[first]['changeset']
+        self.bad_revision = self.build_data[last]['changeset']
```

Comment:
Couldn't you squeeze this all into one line by passing `self.build_data[0]['changeset']` and `self.build_data[-1]['changeset']` as arguments to "reverse_if_hunt_fix"?